src/core/util.cpp

```diff
@@ -756,4 +756,43 @@ namespace mamba
         fs::remove(m_path);
     }
 
+    std::string timestamp(const std::time_t& utc_time)
+    {
+        char buf[sizeof("2011-10-08T07:07:09Z")];
+        strftime(buf, sizeof(buf), "%FT%TZ", gmtime(&utc_time));
+        return buf;
+    }
+
+    std::time_t utc_time_now()
+    {
+        std::time_t now;
+        std::time(&now);
+        gmtime(&now);
```

Comment:
is this doing the same thing twice?

Reply:
it's not, `std::time` get the time since Epoch using the current calendar while `gmtime` converts it to UTC.